apps/designer/app/designer/features/topbar/menu/menu.tsx

```diff
@@ -183,10 +185,12 @@ export const Menu = ({ publish }: MenuProps) => {
           <DropdownMenuItem
             css={menuItemCss}
             onSelect={() => {
-              publish({
-                type: "shortcut",
-                payload: { name: "paste" },
-              });
+              // @todo: need to read from clipboard here. is it possible?
+              //
+              // publish({
+              //   type: "shortcut",
+              //   payload: { name: "paste" },
+              // });
             }}
           >
```

Comment:
Might not be possible to keep copy / paste menu items. 

Webflow seem to support copy, but for paste it says:

<img width="1033" alt="Screenshot 2022-11-22 at 15 34 10" src="https://user-images.githubusercontent.com/825702/203304149-30e0f025-6dfd-41e1-a078-de63683ee3c9.png">

Didn't research it well yet.

Reply:
yeah, probably because it has to be a native copy function, so you can't trigger that copy event, you can only listen to it and add the data to it

Reply:
I've created a separate issue for this #499 . I think we can do this later. Maybe there will be a better support in browsers by then too.